.azure-devops/cleanup.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+# Required for schedule only trigger
+trigger: none
+pr: none
+
+# Run daily at 5 am (Pacific).
+schedules:
+- cron: "0 13 * * *"
+  displayName: 'Daily Cleanup'
+  branches:
+    include:
+    - dev
+  always: true
+
+parameters:
```

Comment:
we should refactor this to pull desired RG and exclusions from our variable library, take a look at  https://github.com/Azure/azure-iot-cli-extension/blob/dev/.azure-devops/create-release.yml#L75 for an example pattern, then the parameters become the name of the variable in the var group that has the respective values

Reply:
but then we need another variable group for this...

ok will do

Reply:
The variable group will be configurable, we can assign the pipeline an existing var group or a new one dedicated for the clean up pipeline (preferred)

Reply:
done